src/chain.h

```diff
@@ -169,7 +169,7 @@ class CBlockIndex
     //! block header
     int nVersion;
     uint256 hashMerkleRoot;
-    uint256 hashReserved;
+    uint256 hashSaplingAnchorEnd;
```

Comment:
`hashFinalSaplingRoot` to match the Sapling specification. If you want `hashFinalSaplingAnchor` instead of `hashFinalSaplingRoot` then convince @daira to update the specification.